content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Verifying image signatures
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+We keep adding support for verifying the integrity and authenticity of contents using the Sigstore project. In this
+post, we focus on verifying container image signatures using the
+new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy
+
+If you don't know how Sigstore works, we recommend reading our
+previous [blog](https://www.kubewarden.io/blog/2022/04/securing-kubewarden-policies/)
+
+## Verify Image Signatures Policy
+
+This policy validates Sigstore signatures for images that match the name provided. If all signature validation pass or
+there is no container that matches the image name, the Pod will be accepted.
```

Comment:
This is conflicting information in one sentence. What exactly are we trying to explain here, please?
As of now, this sentence reads as : If there is NO match for the image name OR the signature validations pass, then the pod gets accepted. Is this the intention?
```suggestion
there is no container that matches the image name, the Pod will be accepted.
```

Reply:
yes, that's correct. a Pod will be accepted if any of this happens:
- All signature validations pass. We will validate signature for all images that match the name provided.
- There is no image match for the name provided. In this case no validation is done and the pod is accepted

Reply:
yeah, this is a bit confusing. I think reusing the README of the policy is better, because it's more clear about what is going to happen.

Reply:
changed in https://github.com/kubewarden/kubewarden.io/pull/87/commits/940ae906d54e4917ebb6aa9e7d1c5bc600cb66ed